.github/workflows/monitoring.yml

```diff
@@ -0,0 +1,79 @@
+name: Teleport Monitoring
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  test:
+    name: Tests
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./packages/teleport-monitoring
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v2
+
+      - name: Set up node
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14.x
+
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+
+      - name: Cache YARN dependencies
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            yarn-
+
+      - run: yarn --no-progress --non-interactive --frozen-lockfile
+
+      - run: yarn lint
+      - run: yarn format
+      - run: yarn test
+      - run: yarn typecheck
+
+  test-e2e:
+    name: Tests E2E
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./packages/teleport-monitoring
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v2
+
+      - name: Set up node
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14.x
+
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+
+      - name: Cache YARN dependencies
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            yarn-
+
+      - run: yarn --no-progress --non-interactive --frozen-lockfile
+
+      - run: |
+          docker run -d --name=teleport_monitoring -p 5432:5432 -e POSTGRES_PASSWORD=password postgres
```

Comment:
I guess the primary benefit of storing the L2 events in a db (vs in-memory) is to persist them across multiple runs of the monitoring job. But right now if the job is restarted, the db is restarted as well. Should we use an external db to persist the data beyond the scope of this CI job?

Reply:
Really, in current tests, we don't sync events. The process is tricked to think that it's fully synced: https://github.com/makerdao/xdomain/pull/23/files#diff-3374d46294cc4947a65c39633c628ff2546a966288c2f8c617280dac7308f87bR32

I think we should avoid writing any tests that would use real testnets as they are very brittle... This test uses fork mode from a concrete block.

Reply:
Oh yeah this is the _test_ CI job. For some reason I got confused and thought this was the CI job that actually runs the monitoring service, but I guess this will actually be run by TechOps and they will have their own setup with their own db.